actors/builtin/miner/miner_state_test.go

```diff
@@ -440,6 +442,252 @@ func TestVestingFundsStore(t *testing.T) {
 	assert.Equal(t, abi.NewTokenAmount(51), vested)
 }
 
+// TODO make top level tests for vested, unvested and then together.
+
+func TestVestingFunds(t *testing.T) {
+	t.Run("Vest all funds in single epoch", func(t *testing.T) {
+		harness := constructStateHarness(t, abi.ChainEpoch(0))
+		vspec := &miner.VestSpec{
+			InitialDelay: 0,
+			VestPeriod:   1,
+			StepDuration: 1,
+			Quantization: 1,
+		}
+		vestStart := abi.ChainEpoch(10)
+		vestSum := abi.NewTokenAmount(100)
+		harness.addLockedFunds(vestStart, vestSum, vspec)
+		amountVested := harness.unlockVestedFunds(vestStart + 2)
+		assert.Equal(t, vestSum, amountVested)
+		assert.Zero(t, harness.s.LockedFunds.Int64())
+		assert.True(t, harness.vestingFundsStoreEmpty())
+		// subsequent calls to unlock return 0
+		amountVested = harness.unlockVestedFunds(vestStart + 2)
+		assert.Zero(t, amountVested)
+
+	})
+	t.Run("Sum of all entries in vesting store is equal to vested sum", func(t *testing.T) {
+		harness := constructStateHarness(t, abi.ChainEpoch(0))
+		vspec := &miner.VestSpec{
+			InitialDelay: 0,
+			VestPeriod:   10,
+			StepDuration: 1,
+			Quantization: 2,
+		}
+		vestStart := abi.ChainEpoch(10)
```

Comment:
This just makes all the numbers below hard to understand. We need at least one test that demonstrates that vestStart is treated correctly, but that test should be very focussed on that. Just use zero elsewhere.